lms/djangoapps/instructor_task/models.py

```diff
@@ -81,13 +82,13 @@ class Meta(object):
 
     def __repr__(self):
         return 'InstructorTask<%r>' % ({
-            'task_type': self.task_type,
-            'course_id': self.course_id,
-            'task_input': self.task_input,
-            'task_id': self.task_id,
-            'task_state': self.task_state,
-            'task_output': self.task_output,
-        },)
+                                           'task_type': self.task_type,
```

Comment:
NIT: Remove this extra spacing.  We don't generally want to push the content further out than the parens.